### PR TITLE
add isMembershipLimitedToOwnersBeta

### PR DESCRIPTION
**Pull request by laujan**

### CODE REVIEW

**Comment by laujan** [comment]
This property in not in V1.0 yet.

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Every team is associated with a [group](../resources/group.md). The group has th
 |discoverySettings|[teamDiscoverySettings](teamdiscoverysettings.md) |Settings to configure team discoverability by others.|
 |webUrl|string (readonly) | A hyperlink that will go to the team in the Microsoft Teams client. This is the URL that you get when you right-click a team in the Microsoft Teams client and select **Get link to team**. This URL should be treated as an opaque blob, and not parsed. |
 |classSettings|[teamClassSettings](teamclasssettings.md) |Configure settings of a class. Available only when the team represents a class.|
+|isMembershipLimitedToOwners|Boolean|If set to `true`, the team is currently in the owner-only team membership state and not accessible by other team members, such as students.|
```
Comment (Lauragra) [inline api-reference/beta/resources/team.md:59]: Please add this new property to the JSON representation as well, and add a changelog entry for the new property.

**Comment by CelesteDG** [comment]
Hello @laujan - I removed the 'ready to merge' label for now per the requested changes from Laura. Please address then when ready you can add back the 'ready to merge' label. Thank you.

File: api-reference/beta/resources/team.md (hunk 1 above)
Comment (laujan) [reply api-reference/beta/resources/team.md:59]: Hi @Lauragra and/or @CelesteDG . Can you please send me a link to the guide for updating the changelog?

Comment (laujan) [reply api-reference/beta/resources/team.md:59]: Hi @Lauragra and/or @CelesteDG. Where can I find the "Id" value for the change? Thanks!

**Pull request merged**